roles/tendrl-ansible.tendrl-server/tasks/tendrl-monitoring-integration.yml

```diff
@@ -28,15 +28,16 @@
     owner: apache
     group: apache
 
-- name: Start carbon-cache service
+- name: Enable carbon-cache service
   service:
     name=carbon-cache
-    state=started
+    enabled=yes
+    daemon_reload=yes
```

Comment:
To use `daemon_reload` here, we need to change ansible module from `service` to `systemd` (line 32).